src/main/resources/swagger.api/schemas/profiles/mappingProfile.json

```diff
@@ -51,6 +51,15 @@
         "MARC"
       ]
     },
+    "fieldsSuppression": {
+      "description":  "Fields to suppress",
```

Comment:
As far as this we should support set of fields does it make sense to have here array of strings?